tests/testsrc/test262.properties

```diff
@@ -697,18 +697,7 @@ built-ins/Date 90/770 (11.69%)
     prototype/setUTCMinutes/not-a-constructor.js {unsupported: [Reflect.construct]}
     prototype/setUTCMonth/not-a-constructor.js {unsupported: [Reflect.construct]}
     prototype/setUTCSeconds/not-a-constructor.js {unsupported: [Reflect.construct]}
-    prototype/Symbol.toPrimitive/hint-default-first-invalid.js
-    prototype/Symbol.toPrimitive/hint-default-first-non-callable.js
-    prototype/Symbol.toPrimitive/hint-default-first-valid.js
-    prototype/Symbol.toPrimitive/hint-invalid.js
-    prototype/Symbol.toPrimitive/hint-number-first-invalid.js
-    prototype/Symbol.toPrimitive/hint-number-first-non-callable.js
-    prototype/Symbol.toPrimitive/hint-number-first-valid.js
-    prototype/Symbol.toPrimitive/hint-string-first-invalid.js
-    prototype/Symbol.toPrimitive/hint-string-first-non-callable.js
-    prototype/Symbol.toPrimitive/hint-string-first-valid.js
-    prototype/Symbol.toPrimitive/length.js
-    prototype/Symbol.toPrimitive/name.js
+    prototype/Symbol.toPrimitive/called-as-function.js
```

Comment:
It looks like we made a bunch of new tests pass, but we also made a test that previously passed fail, from what it looks like. Do you know why this appears to be failing now?

Reply:
Have seen this alteady but there was no time for that. Will check

Reply:
called-as-function.js seems to be a more general problem we have the same with the toJSON prototype.
This function passes in the past because a type error was expected (and there was no toPrimitiveSupport).

Will have another look at this and maybe open an issue for that, but in gerneral it is nothing introduced by this one.